### ISSUE TITLE
Add support for CVX-MCS server and client configuration (for M&E)

### ISSUE THREAD

**Issue by sarunac** [aristanetworks/ansible-avd#1568]
### Enhancement summary

Support for enabling CVX-MCS server and CVX-MCS Client configs for M&E deployment.
Following are the required pieces for this configuration,

- CVX MCS Server
       - Enable cvx 
       - Enable management http 
       - Enable mcs server
       - Input redis password
- CVX MCS Client
        - Enable man cvx
               - Input cvx mcs server ip
        - Enable mcs client
              - Input secondary cvx mcs server ip ( if it is a purple client)
              

### Which component of AVD is impacted

others

### Use case example

MCS is used as a deterministic bandwidth control mechanism for multicast workflows. This is a CVX service, which involves configuration from CVX server and CVX clients . Some of the CVX clients communicate with 2 CVX Servers and these clienst are called Purple clients. Purple clients have an additional configuration that enables them to communicate with 2 CVX servers.

### Describe the solution you would like

It would be good to have support for configuring CVX MCS Server and client,

Following are required configs ,

On CVX Server,

```
cvx
   no shutdown
   !
   service mcs
      redis password 7 070E334ddD1D18
      no shutdown
management api http-commands
   no shutdown
management http-server
   protocol http
```
   
On CVX Client,
```
management cvx
   no shutdown
   server host 10.90.224.188
mcs client
   no shutdown
```

On CVX Purple Client,

```
management cvx
   no shutdown
   server host 172.24.85.239
mcs client
   no shutdown
   !
   cvx secondary blue
      server host 172.24.86.56
``` 

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

**Comment by sugetha24** [comment]
Proposed Data model:
```
### MCS ###
mcs_server:
  cvx: 
    enable: true
  management_api_http:
    enable_http: true
  mcs_server: 
    enable: true
    redis:
      password: 070E334ddD1D18
      encryption: true
    
mcs_client:
  management_cvx:
    shutdown: false
    server_host: 10.90.224.188
  client:
    shutdown: false

purple_client:
  managlement_cvx:
    shutdown: false
    server_host: 172.24.85.239
client:
  shutdown: true
  blue:
    server_host: 172.24.86.56
```

**Comment by sarunac** [comment]
Looks good Sugetha! Thank you for adding the data model.

Q: mcs_server under mcs_server is part of CVX config, I am not sure if the hierarchy here matters.
```
cvx
   no shutdown
   !
   service mcs
      redis password 7 070E33455D1D18
      no shutdown
```

Few Nits: 
Can the 'mcs_server' under mcs_server be 'service_mcs' ?
Under 'client' on 'purple_client' instead of 'blue' can it be 'secondary'. This secondary can be either blue or red cvx server. So a generic name would be better.

**Comment by sugetha24** [comment]
Mcs server:
```
cvx:
    shutdown: < true|false >
    services:
        mcs:
            redis:
                password: 070E334ddD1D18
                encryption: true
            shutdown: < true|false >
 ```

mcs client:
 ```
 management_cvx:
   shutdown: < true|false >
   server_host:
   - 10.90.224.188
   vrf: < vrf_name >
mcs_client:
  shutdown: < true|false >
```
purple client
```
mcs_client:
  shutdown: < true|false >
  cvx_secondary:
  - name: blue
    server_host:
    - 172.24.86.56
```

**Comment by tgodaA** [comment]
Closed in #1682
